packages/maskbook/src/plugins/GoodGhosting/utils.ts

```diff
@@ -30,6 +30,10 @@ export function getNextTimelineIndex(timeline: TimelineEvent[]) {
     return timeline.length - 1
 }
 
+export function getIsEndOfTimeline(timelineIndex: number, timeline: TimelineEvent[]) {
```

Comment:
why function name not is `isEndOfTimeline`

Reply:
Updated the function name :+1: